client/src/elements/molecules/item-list-entry.html

```diff
@@ -13,7 +13,7 @@
   </span>
 
   <delete-button
-    if.bind="user.roles.includes('poweruser')"
+    if.bind="user.roles.includes('poweruser') || user.id == item.conf.createdBy"
```

Comment:
Can you use "===" here. Its saver to use than "==" -> https://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons

Reply:
Just updated this!